canonical-web-and-design/ubuntu.com
# Fix security error pages

## Done

- Render error page based on error code
- create a 404 security error page
- fix date bug for cves that do not have a published date included in the api payload

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Visit http://0.0.0.0:8001/security/CVE-2003-0854 and see that a 404 error page is returned which includes the id you tried to query and a message explaining that a cve with that id does not exist
- Visit http://0.0.0.0:8001/security/CVE-2007-0822 and see that the cve template renders without error (this cve does not have much in its payload, if you notice missing sections it's not an error, they just don't exist)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11290
